index.php

```diff
@@ -32,4 +32,9 @@
     <?php $route['footer'] ? include COMPONENTS . '/footer.php' : null; ?>
   </div>
 </body>
+<style type="text/css">
+  ::-webkit-scrollbar {
+    display: none;
+    }
+</style>
```

Comment:
Dit mag weg, anders is het voor sommige gebruikers verwarrend of ze nog kunnen scrollen of niet.